addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -122,6 +122,10 @@ class MessageList extends Component {
      */
     async adjustFromComponentHints() {
         for (const hint of this.threadViewer.componentHintList) {
+            if (!this.props.hasScrollAdjust) {
+                this.threadViewer.markComponentHintProcessed(hint);
+                continue;
+            }
```

Comment:
to validate this is how we want to handle it cc @alexkuhn

Reply:
The idea is there, but this assumes that all adjustments are on scrolls, which is the case in practice right now but conceptually adjustments could be of many kinds (like adjust with focus, animations, etc.).

That's why I would prefer that each adjustments ignores scroll changes when this prop is set.